src/borg/archiver/delete_cmd.py

```diff
@@ -62,7 +62,7 @@ def build_parser_delete(self, subparsers, common_parser, mid_common_parser):
 
         delete_epilog = process_epilog(
             """
-        This command deletes archives from the repository.
+        This command soft-deletes archives from the repository.
```

Comment:
do we have a glossary somewhere? to avoid having it to explain everywhere, but at least explain it at some defined place.

Reply:
```sh
$ grep -R glossary
docs/internals/data-structures.rst:.. todo:: Clarify terms, perhaps create a glossary.
```

:laughing: 

To be honest, I'm no big fan of glossaries in user docs. They might help for dev docs, but in user docs I'd (naturally I can only speak for myself here...) always rather just google for the word - and if this doesn't bring up meaningful stuff, it needs to be explained "inline". The first result for "soft delete" on Google is perfectly accurate btw:

> Soft delete is a widely used pattern […]. It allows you to mark some records as deleted without actually deleting them […]

---

About "explain it at some defined place": IMHO the best place for that is `delete` and `prune`. We could choose `delete` and just reference it in `prune`, but in the end it's not that big of an explanation. To keep it simple I'd recommend https://github.com/borgbackup/borg/pull/8515#discussion_r1827014820 again. I just did that. What do you think?

Reply:
ok.